packages/localizations/src/pt-BR.ts

```diff
@@ -497,6 +553,12 @@ export const ptBR: LocalizationResource = {
     form_username_invalid_length: 'Nome de usuário deve ter entre 3 e 256 caracteres.',
     identification_deletion_failed: 'Você não pode excluir sua última identificação.',
     not_allowed_access: 'Acesso não permitido.',
+    organization_domain_blocked: 'Este é um provedor de domínio de e-mail bloqueado. Por favor, use um diferente.',
```

Comment:
I went ahead and already added the translations for Brazilian Portuguese + European Spanish